src/main/java/net/mcreator/ui/minecraft/states/JStateLabel.java

```diff
@@ -54,29 +52,26 @@ public JStateLabel(MCreator mcreator, Supplier<List<PropertyData<?>>> properties
 		this.properties = properties;
 		this.otherStates = otherStates;
 
-		setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5));
-
-		label.setEditable(false);

```

Comment:
This was made to allow copying the text inside this field. Blocking input with `setEnabled(false)` instead of `setEditable(false)` is in terms of logic equivalent of replacing it with a label.

Reply:
Editable gives focus which is confusing.

What is a realistic use case of copy?

Reply:
Forget it, forgive me.

Reply:
I was just genuinely curios what is the use case :D